database.js

```diff
@@ -0,0 +1,61 @@
+(function () {
+  const buzzWords = {
+    "buzzWords": []
+  };
+
+  let totalScore = 0;
+
+  const getScore = () => {
+    return totalScore;
+  };
+
+  const updateScore = (index) => {
+    totalScore += buzzWords.buzzWords[index].points;
+  };
+
+  const getBuzzWords = () => {
+    return buzzWords;
+  };
+
+  const addBuzzWord = (word, points) => {
+    let newBuzzWord = {};
+    newBuzzWord.buzzWord = word;
+    newBuzzWord.points = points;
+    buzzWords.buzzWords.push(newBuzzWord);
+    return;
+  };
+
+  const findBuzzWord = (word) => {
+    let index = -1;
+    buzzWords.buzzWords.forEach(buzzWord => {
```

Comment:
So forEach has two extra parameters that it can take in (current, index, array). You could have used the second parameter, which gives you the current index and just returned that.